### PR TITLE
JavaScript capstone project - Your API-based webapp

### PR DESCRIPTION
**Pull request by rubydevi**
**Description:**
This pull request includes the implementation of the home page and comments popup created by @rubydevi and @sadaf-Daneshgar . The following features have been developed:

**Home page:**

- Integration with the selected API to retrieve a list of items and display them on the screen.
- Integration with the Involvement API to show item likes.
- Implementation of the "Like" button functionality to record interactions in the Involvement API and update the screen.
- Implementation of the "Comments" button to open the Comments popup.
- Header and navigation design based on the provided mockup.
- Footer design based on the provided mockup.
- Counters to display the number of items.

**Comments popup:**

- Integration with the selected API to fetch details about the selected item.
- Integration with the Involvement API to display item comments.
- Implementation of the "Comment" button functionality to record data in the Involvement API and update the screen.
- Integration with the selected API to retrieve details about the selected item.
- Integration with the Involvement API to show item reservations.
- Unit tests using Jest have been added to cover the count functions for each item. 

A video presentation of the project's features, including a description of the project, a demo, and showcasing notable code snippets, has also been recorded. The video is a maximum of 5 minutes in length and features all team members discussing the project.

Please review and provide feedback on the implementation.

### CODE REVIEW

File: src/modules/comment-counter.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { getComments } from './api.js';
+
+const displayCommentCount = async (appID, mealID) => {
+  const comments = await getComments(appID, mealID);
+  const commentCount = comments.length;
+  return commentCount;
+};
```
Comment (laiifuu) [inline src/modules/comment-counter.js]: A counter function should look for specific DOM elements and make the counting based on what is actually displayed on the page and not what the API returns.

**Review by laiifuu — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: 

## Hi team :wave: 

Great work :clap: :clap: 
There are some issues that you still need to work on but you are almost there!

so-close

## Highlights
- [x] Use of ES6 and Webpack :heavy_check_mark: 
- [x] Data sent and received using APIs :heavy_check_mark: 
- [x] Use of `async` and `await` :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review :point_down:_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Sadaf & Ruby,  

In my opinion, your project is ready for presentation! Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲  Nice job improving upon test counters!

Cheers..Keep rocking! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by rubydevi** [comment]
Thanks to all the reviewers for your kind reviews.

**Pull request merged**